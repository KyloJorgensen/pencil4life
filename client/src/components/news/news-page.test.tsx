'use strict';

import * as React from 'react';
import { shallow, ShallowWrapper } from 'enzyme';
import NewsPage from './news-page';

let wrapper: ShallowWrapper;

describe('Render NewsPage', () => {
	it('matches the snapshot', () => {
		wrapper = shallow(
			<NewsPage/>
		);
		expect(wrapper).toMatchSnapshot();
	});
});