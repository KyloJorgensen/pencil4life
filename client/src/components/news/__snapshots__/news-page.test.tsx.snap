// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render NewsPage matches the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <UserConsumer />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": [Function],
    },
    "ref": null,
    "rendered": [Function],
    "type": Object {
      "$$typeof": Symbol(react.context),
      "Consumer": [Circular],
      "Provider": Object {
        "$$typeof": Symbol(react.provider),
        "_context": [Circular],
      },
      "_calculateChangedBits": null,
      "_changedBits": 0,
      "_changedBits2": 0,
      "_currentRenderer": null,
      "_currentRenderer2": null,
      "_currentValue": Object {
        "admin": false,
        "checkResetCode": [Function],
        "email": null,
        "firstname": null,
        "forgotPassword": [Function],
        "getUser": [Function],
        "lastname": null,
        "login": [Function],
        "logout": [Function],
        "resetPassword": [Function],
        "signup": [Function],
        "updatePassword": [Function],
        "updateUser": [Function],
        "userAccess": false,
        "username": null,
      },
      "_currentValue2": Object {
        "admin": false,
        "checkResetCode": [Function],
        "email": null,
        "firstname": null,
        "forgotPassword": [Function],
        "getUser": [Function],
        "lastname": null,
        "login": [Function],
        "logout": [Function],
        "resetPassword": [Function],
        "signup": [Function],
        "updatePassword": [Function],
        "updateUser": [Function],
        "userAccess": false,
        "username": null,
      },
      "_defaultValue": Object {
        "admin": false,
        "checkResetCode": [Function],
        "email": null,
        "firstname": null,
        "forgotPassword": [Function],
        "getUser": [Function],
        "lastname": null,
        "login": [Function],
        "logout": [Function],
        "resetPassword": [Function],
        "signup": [Function],
        "updatePassword": [Function],
        "updateUser": [Function],
        "userAccess": false,
        "username": null,
      },
    },
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": [Function],
      },
      "ref": null,
      "rendered": [Function],
      "type": Object {
        "$$typeof": Symbol(react.context),
        "Consumer": [Circular],
        "Provider": Object {
          "$$typeof": Symbol(react.provider),
          "_context": [Circular],
        },
        "_calculateChangedBits": null,
        "_changedBits": 0,
        "_changedBits2": 0,
        "_currentRenderer": null,
        "_currentRenderer2": null,
        "_currentValue": Object {
          "admin": false,
          "checkResetCode": [Function],
          "email": null,
          "firstname": null,
          "forgotPassword": [Function],
          "getUser": [Function],
          "lastname": null,
          "login": [Function],
          "logout": [Function],
          "resetPassword": [Function],
          "signup": [Function],
          "updatePassword": [Function],
          "updateUser": [Function],
          "userAccess": false,
          "username": null,
        },
        "_currentValue2": Object {
          "admin": false,
          "checkResetCode": [Function],
          "email": null,
          "firstname": null,
          "forgotPassword": [Function],
          "getUser": [Function],
          "lastname": null,
          "login": [Function],
          "logout": [Function],
          "resetPassword": [Function],
          "signup": [Function],
          "updatePassword": [Function],
          "updateUser": [Function],
          "userAccess": false,
          "username": null,
        },
        "_defaultValue": Object {
          "admin": false,
          "checkResetCode": [Function],
          "email": null,
          "firstname": null,
          "forgotPassword": [Function],
          "getUser": [Function],
          "lastname": null,
          "login": [Function],
          "logout": [Function],
          "resetPassword": [Function],
          "signup": [Function],
          "updatePassword": [Function],
          "updateUser": [Function],
          "userAccess": false,
          "username": null,
        },
      },
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
